ctags.lua

```diff
@@ -1,5 +1,8 @@
 require('vis')
 
+local positions = {}
+local npos = 0
```

Comment:
There is no need to maintain a separate position counter: `#positions` can be used instead. It's always possible to add a new element with `positions[#positions + 1] = pos`, access the last element using `positions[#positions]` and remove the last element with `table.remove(positions, #positions)`.

Reply:
I agree. First time using lua so I just hacked it together without learning the language. Thanks for the tips